src/popups/DeparturePopup/DeparturePopup.js

```diff
@@ -24,5 +48,9 @@ DeparturePopup.defaultProps = defaultProps;
 
 const composed = compose(withTranslation())(DeparturePopup);
 composed.renderTitle = feat => feat.get('name');
+// Trigerred on popup close with close button only.
+composed.actionOnClose = () => {
```

Comment:
The namimg is a bit confusing since it applies only when we click on the close button.

Reply:
You are right, I renamed it 'onCloseBtClick'.